templates/zerver/help/browse-and-join-streams.md

```diff
@@ -49,7 +49,7 @@ subscribe to (highlighted in red).
 
     ![Subscribe before](/static/images/help/subscribe-before.png)
 
-    A green checkmark will appear next to the stream that you've just
+4. A green checkmark will appear next to the stream that you've just
```

Comment:
I think the numbering is wrong. There is no '1' in the file. This is why it is being rendered as '3' instead of '4'.